conf2sql.pl

```diff
@@ -29,16 +29,23 @@
 
 die "$program: missing --dbdriver option\n" unless defined($opts{'dbdriver'});
 die "$program: missing --dbname option\n" unless defined($opts{'dbname'});
+if (!$opts{'dbdriver'} =~ /sqlite/i) {
```

Comment:
OK.  Please create a separate pull request for each issue found.  That makes it easier to review the pull request, and to modify/merge just the ones that work, while the others are refined.  I cannot and will not accept a single pull request which jumbles together multiple unrelated changes.